### ISSUE TITLE
Parser does not parse wkt's with whitespace after first '('

### ISSUE THREAD

**Issue by ofri-peretz** [mapbox/wellknown#36]
Hey,
I have a service which gives me wkt's with some whitespaces (Do not ask why..) .
After debugging the parse method, found that white() does not match the use case when there is a whitespace after the first '(' character.
Example: 
LINESTRING  ( 30 10, 10 30, 40 40)
It matches first two whitespaces, not matches the whitespace after '('.
Checked for other shapes as well.

**Comment by throberto** [comment]
I spent almost an hour debugging my code until I found this issue.

is this repository still maintained?

**Comment by throberto** [comment]
@tmcw do you still deal with that library?

**Comment by tmcw** [comment]
No, I don't work at Mapbox anymore or have commit rights to this repo.

**Comment by throberto** [comment]
I understand. I will try to apply a correction for this. thanks! 😄
